doc/install/index.rst

```diff
@@ -4,8 +4,64 @@
 Installing MNE-Python
 ======================
 
-.. raw:: html
-   :file: install_type_selector.html
+.. grid:: 2
+
+    .. grid-item-card::
+        :text-align: center
+
+        .. rst-class:: font-weight-bold mb-0
+
+            Standalone installers
+
+        .. rst-class:: card-subtitle text-muted mt-0
+
+            For Beginners
+
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        .. image:: ../_static/mne_installer_macOS.png
+           :alt: MNE-Python Installer
+
+        **New to Python?** Use our standalone installers that include
+        everything to get you started!
+        +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+
+        .. button-ref:: installers
+            :ref-type: ref
+            :color: primary
+            :shadow:
+            :class: font-weight-bold
+
+            |cloud-download-alt| |ensp| Download Installers
```

Comment:
This (the `cloud-download-alt`) currently doesn't render correctly, probably should just get removed for now. Upstream issue: https://github.com/executablebooks/sphinx-design/issues/66

Reply:
Never mind my comments below, then! Feel free to comment out the icons for now then